package.json

```diff
@@ -9,7 +9,7 @@
 		"test": "npm run build && npm run karma",
 		"test-ci": "npm run lint && npm run karma",
 		"test-watch": "onchange -ik -d 500 'source/javascripts/**/*.js.erb' -- npm test",
-		"build": "webpack --bail -p",
```

Comment:
Is `-p` deprecated or some other reason why we should drop it?

Reply:
Yes, it's deprecated

Reply:
It's deprecated